docs/src/Theory/Atmos/EDMFEquations.md

```diff
@@ -143,29 +143,41 @@
 \newcommand{\LatentHeatF}[1]{L_{fus}(#1)}
 ```
 
-# Eddy-Diffusivity Mass-Flux (EDMF) equations
-
-This document is concerned with defining the set of equations solved in the atmospheric turbulence convection model: the EDMF equations. Color-coding is used to indicate:
-
+# Eddy-Diffusivity Mass-Flux (EDMF) parameterization
+
+This document describes the Eddy-Diffusivity Mass-Flux (EDMF)
+parameterization for subgrid-scale (SGS) turbulence and convection.
+The model equations and rationale are based on:
+Tan et al. (2018); Cohen et al. (2020); Lopez-Gomez et al. (2020)
+and Jia et al. (2020).
```

Comment:
This paper is not out yet (not even on arxiv). Could you remove it for now and we can add it back once a preprint is ready (hopefully very soon 🤞 )

Reply:
Will do.